### ISSUE TITLE
Django 3.2: _filter_or_exclude: not enough values to unpack (expected 2, got 0)

### ISSUE THREAD

**Issue by jedie** [radiac/django-tagulous#144]
```
...
  File "/home/jens/repos/PyInventory/.venv/lib/python3.8/site-packages/django/db/models/manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/home/jens/repos/PyInventory/.venv/lib/python3.8/site-packages/django/db/models/query.py", line 985, in complex_filter
    return self._filter_or_exclude(False, args=(), kwargs=filter_obj)
  File "/home/jens/repos/PyInventory/.venv/lib/python3.8/site-packages/tagulous/models/tagged.py", line 119, in _filter_or_exclude
    args, kwargs = args

Exception Type: ValueError at /admin/inventory/itemmodel/
Exception Value: not enough values to unpack (expected 2, got 0)
```
![grafik](https://user-images.githubusercontent.com/71315/136650690-a1732df9-332a-4dca-a17a-5975a9ac20e0.png)

- Django Version: v3.2.8
- Python Version: v3.8.10
- django-tagulous v1.3.0

It's a change in Django 3.1 -> 3.2...

**Comment by nschlemm** [comment]
@jedie @valentijnscholten hope the PR #149 works for you guys as well?

**Comment by valentijnscholten** [comment]
Seems to work, thanks.

**Comment by jedie** [comment]
@nschlemm yes, this fixed the problem. When will it released?

**Comment by nschlemm** [comment]
@radiac can you chime in on the question of a possible release by @jedie ? thx :)

**Comment by valentijnscholten** [comment]
In the mean time you can install tagulous from git+https://github.com/nschlemm/django-tagulous.git@250b88c68507b39da7b5a88798e49805b94e81b8#egg=django-tagulous to get the fix.

**Comment by nschlemm** [comment]
@valentijnscholten while that is a workaround, I do much rather prefer we have an release to rely on - especially with the next updates regarding https://www.djangoproject.com/weblog/2021/dec/07/django-40-released/ already lining up ;)

**Comment by radiac** [comment]
Hi, have seen this and will merge and release asap, but I'm really pushed for time atm - please bear with me and I'll try to get it out in the next week or so.

**Comment by radiac** [comment]
Thanks for bearing with me! Released as 1.3.1.
